blueprints/deploy-config/files/config/deploy.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  //development: {
```

Comment:
Just copied the example deploy.js form the README.  We will need to remember to change this file as well as the README as the API changes